server/src/main/java/io/spine/server/bus/Bus.java

```diff
@@ -207,6 +208,16 @@ public void close() throws Exception {
         return filterChain;
     }
 
+    /**
+     * Obtains the queue of the envelopes.
+     */
+    private DispatchingQueue<E> envelopeQueue() {
```

Comment:
I would easily skip the `envelope` part from its name, as it's the only queue at the moment.

Also, please add some description of what's going on here. And give the `@see` reference to the `DispatchingQueue` class.